components/skus/common/skus_sdk.mojom

```diff
@@ -11,4 +11,5 @@ interface SkusService {
   PrepareCredentialsPresentation(string domain, string path) => (string response);
   CredentialSummary(string domain) => (string response);
   SubmitReceipt(string domain, string order_id, string receipt) => (string response);
+  SubmitReceiptNewOrder(string domain, string receipt) => (string response);
```

Comment:
wondering if maybe `NewOrderFromReceipt` would be a better name?

Reply:
Or `CreateOrderFromReceipt`, which was my initial suggestion.

Reply:
addressed: https://github.com/brave/brave-core/pull/21773/commits/8e295eddc53eec6dc0a649158ee3ecd8ba8df20e